engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -535,6 +481,23 @@ struct shield_of_the_righteous_t : public paladin_melee_attack_t
   }
 };
 
+redoubt_buff_t::redoubt_buff_t( paladin_t* p ) :
```

Comment:
What is that for?

Reply:
Your expire_override doesn't do anything here and what's in the constructor could easily be swapped over to a simple make_buff() like in the previous implementation.